staging/src/k8s.io/apiserver/pkg/server/resourceconfig/helpers_test.go

```diff
@@ -78,10 +79,26 @@ func TestParseRuntimeConfig(t *testing.T) {
 			},
 			err: false,
 		},
+		{
```

Comment:
can we make this resource feature optional? Maybe via flag in the options?

@deads2k what do you have in mind about deprecation?